From af6180b2c918b1993b8f4f6d6be3ec81759832cb Mon Sep 17 00:00:00 2001
From: Fotios Valasiadis <fvalasiad@gmail.com>
Date: Thu, 23 May 2024 01:58:17 +0300
Subject: [PATCH] os: Explicitly include X11/Xmd.h for CARD32 definition to fix
 building on i686

Noticed this after trying to update to xorg-server-xwayland-24.1.0 in void linux https://github.com/void-linux/void-packages/pull/50457

Signed-off-by: Fotios Valasiadis <fvalasiad@gmail.com>
Part-of: <https://gitlab.freedesktop.org/xorg/xserver/-/merge_requests/1561>
---
 os/osdep.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/os/osdep.h b/os/osdep.h
index 0687f568d1..d3c5f4e4f2 100644
--- a/os/osdep.h
+++ b/os/osdep.h
@@ -60,6 +60,7 @@ SOFTWARE.
 #include <limits.h>
 #include <stddef.h>
 #include <X11/Xos.h>
+#include <X11/Xmd.h>
 
 /* If EAGAIN and EWOULDBLOCK are distinct errno values, then we check errno
  * for both EAGAIN and EWOULDBLOCK, because some supposedly POSIX
-- 
GitLab

